### ISSUE TITLE
Master List - PERF / PROG pages

### ISSUE THREAD

**Issue by legoboyvdlp** [legoboyvdlp/A320-family#122]
- [ ] Transition altitude
Should use transition altitude + entered QNH for default.
![image](https://user-images.githubusercontent.com/10434911/80278864-9ac40780-86f1-11ea-812c-4b24ff846a37.png)

We have no way of figuring out what the transition flight level is at the destination airport, so I think we could use the transition altitude entered on INIT A, and apply an offset from the entered QNH to that. Need to confirm this elsewhere, I think. 

- [ ] BARO / RADIO accept input at all times - however, it is only displayed when < 250 miles, else it will always show brackets. I **assume** that when your distance becomes less than 250 miles, your entered BARO / RADIO at that stage becomes visible and you do not need to enter it again.
![image](https://user-images.githubusercontent.com/10434911/80278897-d363e100-86f1-11ea-8829-be7659375014.png)

- [ ] INSERT DEST DATA less than 180 miles, in CRZ or DES page. Amber scratchpad message. If QNH, WIND, or TEMP have not been input.

- [ ] QNH should display amber boxes when distance is less than 180 miles. It should be brackets above 180 miles. It is essential to have QNH input, otherwise managed speed will trip off. Once entered it may be modified, but not cleared.

- [x] VAPP - 100 to 350 knots. REF Smith Thales Manual

- [x] Disallow V1 / VR / V2 input in takeoff phase.

- [x] Minimum THR RED / THR ACC is 400 feet.

- [ ] PROG should show optimum altitude / REC MAX altitude. Ref. graphs at PER CRZ ALT and 22_20_50-10 PROG page.

**Comment by mattmaring** [comment]
DSC-22_20-50-10-28 P 104/136 suggests baro can be changed at any time.

Also, QNH should be amber boxes less than 180 miles.

**Comment by legoboyvdlp** [comment]
That's true, yep :+1: I think it doesn't display anything however, under 250 miles?

Thanks :)

**Comment by mattmaring** [comment]
I plan on adding PROG data as part of my current work
